### PR TITLE
Fix AutoYaST schema in SLE 12 GA

### PR DESCRIPTION
**Pull request by imobach**
Fixes [bsc#954412](https://bugzilla.suse.com/show_bug.cgi?id=954412) for this module. I've added a lot of elements from smb.conf (after checking the documentation).

### CODE REVIEW

File: package/yast2-samba-server.spec [hunk 1]
```diff
@@ -17,7 +17,7 @@
 
 
 Name:           yast2-samba-server
-Version:        3.1.12
+Version:        3.1.13
```
Comment (schubi2) [inline package/yast2-samba-server.spec]: I think it should b 3.1.12.1 as it is GA

**Comment by imobach** [comment]
@schubi2 I wanted to use the same number because I though that SLE-12-GA and SLE-12-SP1 versions were not going to diverge. But I've notice that, although both branches contains the 3.1.12 version, they're slightly different. So I'm using 3.1.12.1 as you suggested and the SLE-12-SP1 will be 3.1.13 when I port this change to SP1.

**Comment by imobach** [comment]
@schubi2 I've finally fixed the version number.

**Comment by schubi2** [comment]
LGTM

**Pull request merged**